src/components/BannerNotifications/README.md

```diff
@@ -1,6 +1,9 @@
 By default it renders nothing:
 ```js
-<BannerNotifications messages={[]} onClick={() => {}} />
+<div>
+	<IconSprite />
```

Comment:
Why does this need to be here?

Reply:
We need to include it on the solo example page https://wikia.github.io/react-design-system/#!/BannerNotifications , in fact we'll need it in all the components that are using `Icon` component now in all the examples. I'll add this there too.

Reply:
@mklucsarits I've added comments.